channels/ve.m3u

```diff
@@ -1,51 +1,61 @@
-#EXTINF:-1 tvg-id="TelevisoradeOriente.ve" tvg-name="Televisora de Oriente" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Televisora de Oriente (480p)
-http://vcp1.myplaytv.com:1935/tvo/tvo/chunklist_w545582271.m3u8
 #EXTINF:-1 tvg-id="TelevisoradeOriente.ve" tvg-name="Televisora de Oriente" tvg-country="VE" tvg-language="Spanish" tvg-logo="" group-title="",Televisora de Oriente (426p)
 http://vcp1.myplaytv.com:1935/tvo/tvo/playlist.m3u8
 #EXTINF:-1 tvg-id="TVFamilia.ve" tvg-name="TV Familia" tvg-country="VE" tvg-language="" tvg-logo="https://i.imgur.com/ixIePRp.png" group-title="",TV Familia (720p)
 https://cdn01.yowi.tv/KPFPGJU8A6/master.m3u8
-#EXTINF:-1 tvg-id="TVFAMILIAVENEZUELA.ve" tvg-name="TV FAMILIA VENEZUELA" tvg-country="VE" tvg-language="" tvg-logo="https://secureservercdn.net/45.40.146.28/1xm.7a5.myftpupload.com/wp-content/uploads/2020/07/New-LOGO-TVF-1-300x300.png" group-title="",TV FAMILIA VENEZUELA (480p)
-https://cdn01.yowi.tv/KPFPGJU8A6/live-1500.m3u8
-#EXTINF:-1 tvg-id="TVFAMILIARHD.ve" tvg-name="TV FAMILIAR HD" tvg-country="VE" tvg-language="" tvg-logo="https://secureservercdn.net/45.40.146.28/1xm.7a5.myftpupload.com/wp-content/uploads/2020/09/loop-TVF.gif?time=1601765650" group-title="",TV FAMILIAR HD (720p)
-https://cdn01.yowi.tv/KPFPGJU8A6/live-2500.m3u8
 #EXTINF:-1 tvg-id="TVVENEZUELA.ve" tvg-name="TV VENEZUELA" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",TV VENEZUELA (480p)
-http://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/chunklist_b1735000.m3u8
+http://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="ValeTV.ve" tvg-name="Vale TV" tvg-country="VE" tvg-language="" tvg-logo="https://www.lyngsat-stream.com/logo/tv/vv/vale_tv_ve.png" group-title="",Vale TV
+http://vcp1.myplaytv.com/valetv/valetv/playlist.m3u8
 #EXTINF:-1 tvg-id="VePlus.ve" tvg-name="Ve Plus" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Ve Plus (480p)
 http://190.122.96.187:8888/http/006
 #EXTINF:-1 tvg-id="Venevision.ve" tvg-name="Venevision" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Venevision
-http://177.52.221.214:8000/play/a0cw/index.m3u8
+https://linkastream.co/headless?url=https://www.youtube.com/watch?v=YNNR0vDH6vA
 #EXTINF:-1 tvg-id="VenezolanadeTelevision.ve" tvg-name="Venezolana de Televisión" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Venezolana de Televisión
 http://177.52.221.214:8000/play/a0cj/index.m3u8
 #EXTINF:-1 tvg-id="VenezolanadeTelevision.ve" tvg-name="Venezolana de Televisión" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Venezolana de Televisión
-https://livestream.streamingconnect.com/hls/vtvweb.m3u8
-#EXTINF:-1 tvg-id="VepacoTV.ve" tvg-name="Vepaco TV" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Vepaco TV [Not 24/7]
-http://vcp1.myplaytv.com:1935/tvepaco/tvepaco/chunklist_w2073389661.m3u8
+https://linkastream.co/headless?url=https://dai.ly/x828i6j
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/129475755-c279ad3f-f691-4de0-bb45-1e5abc0db7de.png)

Reply:
I don't know if it's just me, but not a single link from `linkastream.co` has worked for me in the last month. Is it different for you?

Reply:
Same for me, Keystroke told me that his server doesn't seem to handle the amount of requests it gets. He's finding solutions.

Reply:
Okay.